docs/source/guides/tools/MDLModding.rst

```diff
@@ -304,4 +312,4 @@ The following words correspond to the following:
 ``Crowbar ERROR: The model compiler, "<filename>", does not exist.``
 
 ``Crowbar ERROR: The game's "<filename>" file does not exist.``
-    Mostly happens if you did not properly set up Crowbar, make sure you followed the steps in `Setup Crowbar </guides/tools/mdlmodding.html#setup-crowbar-one-time-only>`__.
\ No newline at end of file
+    Mostly happens if you did not properly set up Crowbar, make sure you followed the steps in `Setup Crowbar </guides/tools/mdlmodding.html#setup-crowbar-one-time-only>`__.
```

Comment:
Okay the diff here looks weird on GitHub but it's just addint a missing newline

Reply:
yeah, dont quite know what happened there, i guess its because its the last line in the file and github is funky but idk.